File: Python/PlusMinus.py
def plusMinus(arr):
    # Write your code here
    positiveCount=0
    negativeCount=0
    zeroCount=0
    arrLength = float(len(arr))
    for val in arr:
        if float(val) > 0:
            positiveCount+=1
        elif float(val) < 0:
            negativeCount+=1
        else:
            zeroCount+=1
    print(positiveCount/arrLength)
    print(negativeCount/arrLength)
    print(zeroCount/arrLength)

File: C#/PlusMinus.cs
public static void plusMinus(List<int> arr)
{
    float positiveCount = 0;
    float negativeCount = 0;
    float zeroCount = 0;
    float arrLength = arr.Count;
    foreach(float val in arr){
        if(val > 0){
            positiveCount++;
        }else if(val < 0){
            negativeCount++;
        }else{
            zeroCount++;
        }
    }
    Console.WriteLine(positiveCount/arrLength);
    Console.WriteLine(negativeCount/arrLength);
    Console.WriteLine(zeroCount/arrLength);

}